src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1467,4 +1466,10 @@ Please specify one of them using the `--source` option to proceed.</value>
     <value>Archive scan detected malware; proceeding due to --force</value>
     <comment>{Locked="--force"}</comment>
   </data>
+  <data name="NoUpgradeArgumentDescription" xml:space="preserve">
+    <value>Do not try to upgrade any installed version during install operation</value>

```

Comment:
>>Do not try to upgrade any installed version during install operation [](http://example.com/codeflow?start=10&length=69)

Nit: This sounds like a warning message rather than an argument description.

How about: "Skips upgrade if an installed version already exists"